languages/go/exercises/concept/strings/.docs/introduction.md

```diff
@@ -0,0 +1,19 @@
+A `string` in Go is an immutable sequence of bytes. Strings may contain arbitrary data, including bytes with values 0, but usually they contain human-readable text.
+Text strings are conventionally interpreted as UTF-8 encoded sequence of Unicode code points (runes) which will be explained in a future exercise.
+A `string` value can be written as a string literal, a sequence of bytes enclosed in double quotes:
+
+```go
+s := "Hello World!"
+```
+
+In Go floating point values are convenietly printed with Printf's verbs: %g (compact representation), %e (exponent) or %f (non exponent). All three verbs allow field width and numberic position to be controlled.
+
+```go
+f := 4.3242
+fmt.Printf("%.4", f)
```

Comment:
Is the `fmt` package being introduced in this exercise? If so, it might be worth explicitly mentioning it. @tehsphinx?

Reply:
Yes, you are right. `fmt` has not been introduced.

@oanaOM: Could we add a small part on the `fmt` package to the introduction? I think this is the correct exercise to to this.

We will however have to iron out a lot of these inconsistencies later on anyway. Wouldn't consider this blocking the merge.

Reply:
Sure thing! I'll mention about this package as well.